src/topology/locally_constant/algebra.lean

```diff
@@ -151,4 +153,33 @@ function.injective.distrib_mul_action coe_fn_add_monoid_hom coe_injective (λ _
 instance [semiring R] [add_comm_monoid Y] [module R Y] : module R (locally_constant X Y) :=
 function.injective.module R coe_fn_add_monoid_hom coe_injective (λ _ _, rfl)
 
+section algebra
+
+variables [comm_semiring R] [semiring Y] [algebra R Y]
+
+/-- Constant functions as a `ring_hom`.
+
+See also `continuous_map.C`. -/
+@[simps] def C : R →+* locally_constant X Y :=
+{ to_fun    := λ c, const X $ algebra_map R Y c,
+  map_one'  := by { ext, exact (algebra_map R Y).map_one, },
+  map_mul'  := by { intros, ext, exact (algebra_map R Y).map_mul _ _, },
+  map_zero' := by { ext, exact (algebra_map R Y).map_zero, },
+  map_add'  := by { intros, ext, exact (algebra_map R Y).map_add _ _, }, }
```

Comment:
I suspect you can define this more easily as `ring_hom.comp { to_fun := const, ... } (algebra_map R Y)`

Reply:
Right, yes!

And in fact now that I think about it, `C` is a bit pointless since it's just another way to say `algebra_map R (locally_constant X Y)`.